### PR TITLE
Show-Karma - Properly handle completion of topics

### PR DESCRIPTION
**Pull request by vexx32**
﻿# PR Summary

Show-Karma wasn't expecting completion at all and was completely mishandling the prompt parameter call. This PR resolves that issue and tidies up some surrounding and related test code for Show-Karma and Get-Karma.

## Context

See #241 where the issue arose.

## Changes

- Add check in Show-Karma for completion of a topic.
- Adjust problematic tests & remove unused mocks.
- Appropriately adjust parameters used when splatting to Show-MeditationPrompt.
- Refactor Get-Karma tests.
- Add launch.json to simplify debugging down the line.

## Checklist

- [X] Pull Request has a meaningful title.
- [X] Summarised changes.
- [X] Pull Request is ready to merge & is not WIP.
- [x] Added tests / only testable interactively.
  - Make sure you add a new test if old tests do not effectively test the code changed.
- [ ] Added documentation / opened issue to track adding documentation at a later date.

### CODE REVIEW

**Comment by vexx32** [comment]
@seeminglyscience would you mind giving this a once over to make sure I've not done anything else dumb here? 😂

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            "cwd": ""
+        },
+        {
+            "name": "PowerShell Attach Interactive Session Runspace",
+            "type": "PowerShell",
+            "request": "attach",
+            "processId": "current"
+        }
+    ]
+}
```
Comment (SeeminglyScience) [inline .vscode/launch.json]: New line at EOF

Comment (vexx32) [reply .vscode/launch.json]: Thanks! Haha

Comment (vexx32) [reply .vscode/launch.json]:
```suggestion
}

```

**Pull request merged**